checker/build.gradle

```diff
@@ -584,7 +584,15 @@ task wpiManyTests(group: "Verification") {
                 throw new GradleException("Failure: WPI produced empty typecheck file " + filename)
             }
             file.eachLine { line ->
-                if (line.startsWith("Running ") || line.startsWith("warning: [path]")
+                if (
+                        // Ignore the line that WPI echoes with the javac command being run.
+                        line.startsWith("Running ")
+                        // Warnings about bad path elements aren't related to WPI and are ignored.
+                        || line.startsWith("warning: [path]")
+                        // WPI issues this warning when no stub files are generated (i.e. when no annotations are
+                        // inferred). This is the case for some of the tests, so it should be ignored.
+                        || line.startsWith("warning: Did not find annotation file")
```

Comment:
#4166 eliminates that warning, so this condition is no longer necessary when #4166 is merged.
If this is merged first, then please pull master into #4166 and delete this line and its comment.
If #4166 is merged first, then please delete this line and its comment.